templates/zerver/help/mute-a-stream.md

```diff
@@ -1,6 +1,8 @@
 # Mute a stream
 
-You can mute streams to reduce the amount of notifications you receive.
+If you would like to stop receiving notifications from a certain stream, you can
+choose to mute it. You can also [mute individual topics](/help/mute-a-topic) for
+more granularity.
```

Comment:
I'm not sure `granularity` is the right word here, maybe just leave `you can mute individual topics`?